cinema/models.py

```diff
@@ -35,12 +39,23 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance, filename):
+    ext = os.path.splitext(filename)[1]
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}{ext}"
+    return os.path.join("uploads/movies/", filename)
```

Comment:
```suggestion
def movie_image_file_path(instance, filename):
    ext = os.path.splitext(filename)[1]
    filename = f"{slugify(instance.title)}-{uuid.uuid4()}{ext}"
    return os.path.join("uploads", "movies", filename)
```

Read how this method works